lib/types/terminus.js

```diff
@@ -1,17 +1,40 @@
 'use strict';
 var types = require('../types'),
+    errors = require('../errors'),
     Terminus = module.exports = {};
 
+function terminusDurability(value) {
```

Comment:
well I've been trying to be consistent in terms of supporting the path where you might be using our constants still (like, the whole codebase), while also supporting string versions. But yeah I'll default it to `none` in the field definition